audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py

```diff
@@ -218,16 +218,16 @@ def preprocess_dataset(files):
 input_shape = tf.squeeze(specs[0]).shape
 
 model = models.Sequential([
-    layers.Input(shape=input_shape),
-    preprocessing.Resizing(32, 32),
-    norm_layer,
-    layers.Conv1D(32, 3, activation='relu'),    # changed all these to 1D
-    layers.Conv1D(64, 3, activation='relu'),
-    layers.MaxPooling1D(),
-    layers.Dropout(0.25),
-    layers.Flatten(),
-    layers.Dense(128, activation='relu'),
-    layers.Dropout(0.5),
+    layers.Input(shape=[None, 1000]),
+    # preprocessing.Resizing(32, 32),
+    # norm_layer,
```

Comment:
I just got rid of these to remove potential places for the tensorflow errors to occur. Add them back in when you are ready